### PR TITLE
[ADD] accounting/localization: l10n_pe functional documentation

### PR DESCRIPTION
**Pull request by ren-odoo**
Functional documentation for the Peruvian Localization: 

- Intro
- Configuration
- Tesiting and usage

### CODE REVIEW

**Comment by ren-odoo** [comment]
Hi @jcs-odoo 

Can  you help to take a look at this PR please, we're planning to merge l10n_pe_edi tomorrow (monday 25th), so the sooner we can published the Functional documentation the better. 
 
Note: I got a warining in line 128 due to the table format, but I followed the formating instructions from RST cheat sheet, let me know what change is needed. 

Thank you in advance.

File: accounting/fiscal_localizations/localizations/peru.rst [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+
+.. image:: media/peru_25_credit_note_document.png
+   :align: center
+   :alt: Credit Note document type
+
+To finish the workflow please follow the instructions on 
+`this section <https://www.odoo.com/documentation/user/14.0/accounting/receivables/customer_invoices/credit_notes.html>`_.
+
+.. note::
+   The EDI workflow for the SUNAT in the same way as the invoices.
```
Comment (jcs-odoo) [inline accounting/fiscal_localizations/localizations/peru.rst:527]: I think a part of the sentence is missing.

File: accounting/fiscal_localizations/localizations/peru.rst [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+To finish the workflow please follow the instructions on 
+`this section <https://www.odoo.com/documentation/user/14.0/accounting/receivables/customer_invoices/credit_notes.html>`_.
+
+.. note::
+   The EDI workflow for the SUNAT in the same way as the invoices. 
+
+
+Debit Notes
+------------
+As part of the Preuvian localization, besides creating credit notes from an existing document
```
Comment (jcs-odoo) [inline accounting/fiscal_localizations/localizations/peru.rst:532]:
```suggestion
As part of the Peruvian localization, besides creating credit notes from an existing document 
```

**Comment by jcs-odoo** [comment]
I forgot to ask:
is the localization only available on odoo 14?
Otherwise, would this doc be also valid for older versions?

**Comment by ren-odoo** [comment]
> I forgot to ask:
> is the localization only available on odoo 14?
> Otherwise, would this doc be also valid for older versions?

Yes, this module will be released, only on V14.

**Comment by ren-odoo** [comment]
> * accounting/fiscal_localizations/overview/localizations_list.rst

Hi @jcs-odoo  thank you for the feedback 

I have addressed all the comments and commited your suggestions as well. 

Regarding the changes on accounting/fiscal_localizations/overview/localizations_list.rst, the Colombian ones apply on V13 as well so I remove them from here to latter create a PR just for that on V13. 

Let me know if there is anything else. 

Thank you.

**Review by jcs-odoo — APPROVED** [review]

**Review by jcs-odoo — COMMENTED** [review]
just a few comments :)

File: accounting/fiscal_localizations/localizations/peru.rst [hunk 3]
```diff
@@ -0,0 +1,583 @@
...
+  <https://cpe.sunat.gob.pe/aliados/ose#:~:text=El%20Operador%20de%20Servicios%20Electr%C3%B3nicos%20(OSE)%20es%20qui%C3%A9n%20se%20encarga,otro%20documento%20que%20se%20emita>`_.
+- **CDR**: Receipt certificate (Constancia de Recepción). 
+- **SOL Credentials**: User and password provided by the SUNAT and grants access to Online 
+  Operations systems. 
+
+
+Signature Provider
+~~~~~~~~~~~~~~~~~~
+
+As part of  the requirements for Electronic Invoice is Peru, your company needs to select a
```
[the remark this answers is not in the document]
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/peru.rst]: Fixed in last commit.

File: accounting/fiscal_localizations/localizations/peru.rst [hunk 4]
```diff
@@ -0,0 +1,583 @@
...
++---------+-----+
+| 20,000  | 440 |
++---------+-----+
+
+The credits are consumed per each document that is sent to the OSE. 
+
+.. important::
+   If you have a validation error and the document needs to be sent one more time, an additional 
+   credit will be charged. Therefore, it is paramount that you verify all information is correct 
+   before sending your document to the OSE.
```
[the remark this answers is not in the document]
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/peru.rst]: Fixed in last commit.

File: accounting/fiscal_localizations/localizations/peru.rst [hunk 5]
```diff
@@ -0,0 +1,583 @@
...
+available:
+
+- **l10n_pe**: Adds accounting features for the Peruvian localization, which represent the minimal
+  configuration required for a company to operate in Peru and under the SUNAT regulations and 
+  guidelines. The main elements included in this module are: Chart of account, taxes, 
+  document types. 
+
+- **l10n_pe_edi**: includes all technical and functional requirements to generate and validate 
+  Electronic Invoice, based on the SUNAT specification to create and process valid electronic 
+  documents, for more technical detail you can access to
```
[the remark this answers is not in the document]
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/peru.rst]: Fixed in last commit.

**Pull request merged**